packages/terra-clinical-item-display/src/ItemDisplay.jsx

```diff
@@ -51,11 +60,13 @@ const ItemDisplay = ({
   isTruncated,
   isDisabled,
   icon,
+  iconAlignment,
   ...customProps
 }) => {
   const componentClassNames = cx([
     'item-display',
     { 'is-disabled': isDisabled },
+    { [`icon-${iconAlignment}`]: iconAlignment },
```

Comment:
Should we even add this if there is no `icon`?

Reply:
Thanks @naveen, was thinking about it backwards and removed `icon` instead of `iconAlignment`. Fixed: https://github.com/cerner/terra-clinical/pull/490/commits/c86aab122395aef6137dbf691f69476d0949ab57#diff-7a7f3d6691e06f96790e846013f347ff